conbench/app/_plots.py

```diff
@@ -150,12 +150,18 @@ def simple_bar_plot(benchmarks, height=400, width=400, vbar_width=0.7):
     return p
 
 
-def _should_format(data, unit):
+# Using `Optional[float]` here only because of BenchmarkResult.mean being
+# nullable as of today.
+def _should_format(floats: list[Optional[float]], unit):
```

Comment:
this function is tough -- purpose is still cryptic to me. unit tests are there, importance of those not clear. cost of change was already too big. good re-write candidate.